app/uk/gov/hmrc/gform/validation/ValidationService.scala

```diff
@@ -557,25 +546,25 @@ class ComponentsValidator(
       case "true" :: Nil =>
         List(
           Monoid[ValidatedType].combine(
-            validateRequiredField("street1", "Address line 1")(addressValueOf("street1")),
+            validateRequiredField("street1", "line 1 ")(addressValueOf("street1")),
```

Comment:
```
"line 1 "
```
can be replaced by 
```
"line 1"
```
given the change I mentioned previously.

Reply:
not sure what I was thinking! I removed the spaces and put them into the join instead.